src/Sentry/Protocol/Envelopes/Envelope.cs

```diff
@@ -445,6 +445,21 @@ internal static Envelope FromClientReport(ClientReport clientReport)
         return new Envelope(header, items);
     }
 
+    [Experimental(DiagnosticId.ExperimentalFeature)]
```

Comment:
```suggestion
    // TODO: This is temporary. We don't expect single log messages to become an envelope by themselves since batching is needed
    [Experimental(DiagnosticId.ExperimentalFeature)]
```